packages/dashboard/src/components/robots/robot-page.tsx

```diff
@@ -38,14 +62,14 @@ export function RobotPage() {
     return resp.data.items;
   }, [fleetsApi, page]);
 
-  React.useEffect(() => {
-    fetchVerboseRobots();
-  }, [fetchVerboseRobots]);
+  useAutoRefresh(fetchVerboseRobots, 2000, autoRefresh);
```

Comment:
This fetches a new set of data, overriding any of the current data. So this will cause rows to jump around, items in a page to change etc. I'm not sure if this is desirable for users, and also this is different from how it works in the tasks page, we should keep the behaviour of "auto refresh" consistent.